metadata.json

```diff
@@ -21,7 +21,6 @@
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "10",
```

Comment:
At least test suite related to this OS :
https://github.com/voxpupuli/puppet-fail2ban/blob/master/spec/classes/init_spec.rb#L33

I did not check all the module.

Reply:
Right, I will have another look